org-roam.el

```diff
@@ -648,7 +648,8 @@ This needs to be quick/infrequent, because this is run at
     (when (and (or redisplay
                    (not (eq org-roam--current-buffer buffer)))
                (eq 'visible (org-roam--current-visibility))
-               (buffer-local-value 'buffer-file-truename buffer))
+               (buffer-local-value 'buffer-file-truename buffer)
+               (org-roam-cache-initialized))
```

Comment:
just a small nit, can be name helper predicate functions like `org-roam--cache-initialized-p`